gallery/package.json

```diff
@@ -55,7 +55,9 @@
     "@nivo/bar": "^0.80.0",
     "@nivo/line": "^0.80.0",
     "@nivo/pie": "^0.80.0",
+    "@nivo/tooltip": "^0.80.0",
```

Comment:
I'm noticing these are in `devDependencies`.  None of the nivo stuff should be dev dependencies.

Reply:
Updated here 7e54782ca0f389f9be6c51b539e8ee7b49bec6af. Thanks.